index.html

```diff
@@ -11,9 +11,67 @@
 </head>
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <a class="logo" href="index.html"><img src="/img/karma-logo.svg" alt="karma logo"></a>
+        <nav>
+            <ul class="nav-list">
+                <li class="nav-items" ><a class="nav-links" href="index.html">Meet Karma</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">How it Works</a></li>
+                <li class="nav-items" ><a class="nav-links" href="store.html">Store</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Blog</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Help</a></li>
+                <li class="nav-items" ><a class="nav-links" href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <h2>Introducing Karma</h2>
+            <h5>Bring WiFi with you, everywhere you go.</h5>
+            <button><a href="#">Learn More</a></button>
+        </section>
+        <section>
+            <h2>
+                Everyone needs a little Karma.
+            </h2>
+            <article>
+                <div class="context">
+                    <img src="/img/icon-devices.svg" alt="device    icon">
+                    <h5>Internet for all devices</h5>
+                </div>
+                <div class="context">
+                    <img src="/img/icon-coffee.svg" alt="coffee     icon">
+                    <h5>Boost your productivity</h5>
+                </div>
+                <div class="context">
+                    <img src="/img/icon-refill.svg" alt="refill     icon">
+                    <h5>Pay sd You Go</h5>
+                </div>    
+            </article>
+        </section>
+        <section class="continer_ads">
+<div class="container_photo"></div>
+<div class="container_context">
+    <p>"Wherever I am ,I just dont worry about my connection anymore"</p>
+    <a href="store.html">Get karma today</a>
+</div>
 
+
+
+
+        </section>
```

Comment:
I'd again make sure we're minimising the whitespace here, and make sure we're following best code formatting practices.